app/pages/learn/01_tutorial/01_your-first-java-app/01_getting-started-with-java.md

```diff
@@ -259,39 +259,39 @@ Your shell should print the complete path to the `java` executable file in the `
 <a id="compiling">&nbsp;</a>
 ## Compiling your First Class
 
-Once you have properly set up your JDK; the `JAVA_HOME` variable and the `PATH` variable, you are ready to compile your first class.
+Once you have properly set up your JDK, the `JAVA_HOME` variable, and the `PATH` variable, you will be ready to compile your first class.
 
 All the commands you will be typing now should be typed in the same prompt as the one you used to set up these two variables.
 
 ### Compiling and Running Your First Java Program
 
-Whether you followed the Windows, the Linux or the macOS path, the remaining is the same.
+The following instructions are the same regardless of if you followed the the Windows, the Linux or the macOS path.
 
-1. Change to the directory where you saved your first class `MyFirstClass.java`. You can check that you are in the right directory by typing `dir`. It will show you the files you have in this directory. You should see your `MyFirstClass.java` file.
-2. Check that your compiler is accessible from this directory by typing the following. This command is the same whether you are on Windows or Linux.
+1. Change to the directory where you saved your first class, `MyFirstClass.java`. You can check that you are in the right directory by typing `dir` (for Windows) `ls` (for macOS or Linux). This command will list the files you have in this directory. You should see your `MyFirstClass.java` file in that list.
```

Comment:
I think there should be some sort of separation/connector between `dir` and `ls`, for example
```
You can check that you are in the right directory by typing `dir` (on Windows) or `ls` (for macOS or Linux).
```